3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -96,6 +96,17 @@ enrich {
       brokers = "{{kafkaBrokers}}"
       # Number of retries to perform before giving up on sending a record
       retries = 0
+      # The kafka producer has a variety of possible configuration options defined at
+      # https://kafka.apache.org/documentation/#producerconfigs
+      #producerConf {
```

Comment:
we might want to specify the configuration that are already used so that there is no override, wdyt?

Reply:
Good point. However, since some values are set from other values in the configuration file, I would suggest only specifying the remaining (fixed) values and referring to the other locations.
Like this: https://gist.github.com/0xE282B0/d91b8fac239c360cfd4c8e58901f87b0#file-config-hocon-sample-L94

Did I get you right?

Reply:
I'm not sure I fully understand what you mean, could you give an example? either here or as an additional commit :+1:

Reply:
I've added all values that are hardcoded in the `createProperties(...)` methods of the [source](https://github.com/snowplow/snowplow/blob/master/3-enrich/stream-enrich/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/KafkaSource.scala#L125) and [sink](https://github.com/snowplow/snowplow/blob/master/3-enrich/stream-enrich/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sinks/KafkaSink.scala#L53).

For the other values which are collected from other parts of the config I added a comment. Maybe it's a bit to crowdy.